backend/src/test/java/de/bund/digitalservice/ris/caselaw/domain/DocumentationUnitStatusServiceTest.java

```diff
@@ -92,6 +96,49 @@ void testUpdate_withDocumentNumberAndDocumentationUnitNotFound_shouldNotSaveASta
                 .withError(true)
                 .documentationUnit(documentationUnitDTO)
                 .build());
+
+    verify(historyLogService, times(1))
+        .saveHistoryLog(
+            documentationUnitDTO.getId(),
+            user,
+            HistoryLogEventType.STATUS,
+            "Status geändert: Unveröffentlicht → Veröffentlicht");
+  }
+
+  @Test
+  void
+      testUpdate_withDocumentNumberAndDocumentationUnitFoundWithExistingStatus_shouldNotWriteHistoryLogIfStatusUnchanged()
```

Comment:
I don't quite get, why this name should be so long... Does it add any important info?
```suggestion
      testUpdate_withExistingStatus_shouldNotWriteHistoryLogIfStatusUnchanged()
```